utils/resource/usage.go

```diff
@@ -178,8 +178,9 @@ func (m *manager) update(diskPath string, frequency, cpuHalflife, diskHalflife t
 
 		availableBytes, getBytesErr := storage.AvailableBytes(diskPath)
 		if getBytesErr != nil {
-			m.log.Debug("failed to lookup resource",
+			m.log.Verbo("failed to lookup resource",
 				zap.String("resource", "system disk"),
+				zap.String("path", diskPath),
```

Comment:
I wanted to include the path here as well because I was initially confused why I was also seeing this error (was running with `--db-type=memdb`)